docs/src/examples/IEGT_example1.jl

```diff
@@ -0,0 +1,91 @@
+# # Foundations of Bilevel Programming: Example 9
+# This example is from the book Investment in Electricity Generation and Transmission. [url](https://www.springer.com/gp/book/9783319294995)
+
+# Bold point(s): using Dualof() in case we have information on the dual variable(s) of the lower level problem. 
+
+
+# Model of the problem
+# First level
+# ```math
+# \min 40000x,\\
+# \notag s.t.\\
+# 0 \leq x \leq 250,\\
+# ```
+# Second level
+# ```math
+# \min 10y_1 + 12y_2 + 15y_3,\\
+# \notag s.t.\\
+# y_1 + y_2 + y_3 = 200,\\
+# y_1 \leq x,\\
+# y_2 \leq 150,\\
+# y_3 \leq 100,\\
+# y[i] \geq 0, \forall i \in I\\
+# ```
+
+# Model with doul variable
+# ```math
+# \min 40000x + 8760*(10y_1-\lambda * y_1),\\
```

Comment:
Where does the 8760 come from? What is `\lambda`?

Reply:
lambda is the dual variable of one of the second-level problem.

Reply:
8760 is the number of hours in the year.
that is part of the formulation in the paper. as @odow said we should add a tiny bit of context. Explaining physical meaning of variables and constraints....